nipype/interfaces/dipy/base.py

```diff
@@ -211,3 +211,29 @@ def _list_outputs(self):
                      "_run_interface": _run_interface,
                      "_list_outputs:": _list_outputs})
     return newclass
+
+
+def get_dipy_workflows(module):
+    """Search for DIPY workflow class.
+
+    Parameters
+    ----------
+    module : object
+        module object
+
+    Returns
+    -------
+    l_wkflw : list of tuple
+        This a list of tuple containing 2 elements:
+        Worflow name, Workflow class obj
+
+    Examples
+    --------
+    >>> from dipy.workflows import align  # doctest: +SKIP
+    >>> get_dipy_workflows(align)  # doctest: +SKIP
+
+    """
+    return [(m, obj) for m, obj in inspect.getmembers(module)
+            if inspect.isclass(obj) and
+            issubclass(obj, module.Workflow) and
+            m not in ['Workflow', 'CombinedWorkflow']]
```

Comment:
As in https://github.com/nipy/dipy/pull/1800, I suggest that you set this list to be a module-level constant. 

BTW: the fact that you are using this same code there suggests that this might be a pattern worth including in the dipy library code, instead of here.